mozilla/foundation.mozilla.org
# Richtext product page fields

Closes #7694 
Related PRs/issues #

**Link to sample test page**: https://foundation-s-richtext-p-hhzunw.mofostaging.net/en/privacynotincluded/between-would-deal-effort-table-ability-technology-evidence/

**Steps to test:**

1. Please visit the test page linked above
2. Scroll down to the following fields, and check if they are rich text:
- [ ]  What is the company’s known track record of protecting users’ data? (Privacy Tab)
- [ ] How does the company use this data? (Privacy Tab)
- [ ] How can you control your data? (Privacy Tab)
- [ ] General AI description/helptext (AI tab)
- [ ] Manages security vulnerabilities Description/helptext (security tab)
3. If everything is looking/behaving correctly, testing is complete! 

Please note that since Faker and Factory run after migrations, the rest of the pages are not going to render the text styled as the test page.
When this gets merged into staging/prod, the new migration will take all existing pages and update them as expected.



### Screenshots of fields on review app:
<img width="892" alt="image" src="https://user-images.githubusercontent.com/18314510/140003037-c8124faa-566d-4cd5-a1f2-df25cd1ad96d.png">

<img width="892" alt="image" src="https://user-images.githubusercontent.com/18314510/140003059-24dbf546-4d5b-47e2-a9a3-fa97d6ae3bf1.png">

<img width="892" alt="image" src="https://user-images.githubusercontent.com/18314510/140003082-056da29e-876d-413d-8c77-389576f9f54e.png">






## Checklist
**Changes in Models:**
